optuna/multi_objective/__init__.py

```diff
@@ -1,6 +1,10 @@
 # TODO(nabenabe0928): Come up with any ways to remove this file.
 # NOTE(nabenabe0928): Discuss when to remove this class.
+migration_url = "https://github.com/optuna/optuna/discussions/5573"
 raise ModuleNotFoundError(
-    "`optuna.multi_objective` was removed at v4.0.0. "
-    "Please update your code or downgrade your Optuna version."
+    "\nThe features in ``optuna.multi_objective`` were integrated with the"
+    "\nsingle objective optimization API and ``optuna.multi_objective`` were"
+    "\ndeleted at v4.0.0. Please update your code based on the migration guide"
+    f"\nat {migration_url}"
+    "\nor downgrade your Optuna version."
```

Comment:
```suggestion
    "\nThe features in `optuna.multi_objective` were integrated with the"
    "\nsingle objective optimization API and `optuna.multi_objective` were"
    "\ndeleted at v4.0.0. Please update your code based on the migration guide"
    f"\nat {migration_url}"
    "\nor downgrade your Optuna version."
```

This is a very nit-picky, but a single backtick character is more suitable since this error text is not rendered as a reStructuredText.

```python
>>> import optuna.multi_objective
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/cbata/go/src/github.com/optuna/optuna/optuna/multi_objective/__init__.py", line 4, in <module>
    raise ModuleNotFoundError(
ModuleNotFoundError:
The features in ``optuna.multi_objective`` were integrated with the
single objective optimization API and ``optuna.multi_objective`` were
deleted at v4.0.0. Please update your code based on the migration guide
at https://github.com/optuna/optuna/discussions/5573
or downgrade your Optuna version.
```

Reply:
Thank you for the suggestion, I applied your suggestion!